src/markitdown/_markitdown.py

```diff
@@ -837,6 +838,88 @@ def _get_mlm_description(self, local_path, extension, client, model, prompt=None
         return response.choices[0].message.content
 
 
+class ZipConverter(DocumentConverter):
+    """Converts ZIP files to markdown by extracting and converting all contained files."""
```

Comment:
Can you please expand the docstring with the example format it generates?

Reply:
Done, is that better?

Reply:
Amazing 🤌🤌🤌